stdlib/InteractiveUtils/src/macros.jl

```diff
@@ -4,7 +4,7 @@
 
 import Base: typesof, insert!
 
-separate_kwargs(args...; kwargs...) = (args, kwargs.data)
+separate_kwargs(args...; kwargs...) = (args, getfield(kwargs, :data))
```

Comment:
```suggestion
separate_kwargs(args...; kwargs...) = (args, kwargs)
```

Reply:
Looks like this is not just used in InteractiveUtils itself, so the conversion to a NamedTuple is actually relevant.